draft-ietf-tls-subcerts.md

```diff
@@ -95,45 +97,64 @@ clarity, we will refer to the certificate issued by the CA as a "certificate",
 or "delegation certificate", and the one issued by the operator as a "delegated
 credential".
 
+## Change Log
+
+(\*) indicates changes to the wire protocol.
+
+draft-02
+
+  * Drop support for TLS 1.2.
+
+  * Add the protocol version and credential signature algorithm to the
+    Credential structure. (*)
+
+  * Specify undefined behavior in a few cases: when the client receives a DC
+    without indicated support; when the client indicates the extension in an
+    invalid protocol version; and when DCs are sent as extensions to
+    certificates other than the end-entity certificate.
+
+
 # Solution Overview
 
 A delegated credential is a digitally signed data structure with two semantic
-fields: a validity interval, and a public key (with its associated algorithm).
-The signature on the credential indicates a delegation from the certificate that
-is issued to the TLS server operator. The secret key used to sign a credential
-is presumed to be one whose corresponding public key is contained in an X.509
-certificate that associates one or more names to the credential.
+fields: a validity interval and a public key (along with its associated
+algorithm).  The signature on the credential indicates a delegation from the
+certificate that is issued to the TLS server operator.  The secret key used to
+sign a credential is presumed to be one whose corresponding public key is
```

Comment:
instead of `is presumed to be`, maybe `is one whose corresponding public key....`

Reply:
Done.